sass/main.scss

```diff
@@ -225,3 +229,38 @@ img {
   }
 
 }
+
+.slide_members {
+  @extend .section;
+
+  h1 {
+    padding-bottom: 32px;
+  }
+
+  .member {
```

Comment:
I'd avoid nesting this rule unless you really need to. Just like extends, nesting can make unintentionally crazy selectors in the outputted CSS.